example/lib/widgets/drawer/menu_drawer.dart

```diff
@@ -61,6 +63,11 @@ class MenuDrawer extends StatelessWidget {
                   textAlign: TextAlign.center,
                   style: TextStyle(fontSize: 14),
                 ),
+                const Text(
+                  _isWASM ? 'Running with WASM' : 'Running without WASM',
+                  textAlign: TextAlign.center,
+                  style: TextStyle(fontSize: 14),
+                ),
```

Comment:
idea: only display this text when running on web, could cause confusion when running on other platforms.

Reply:
Good idea, done.